\hypertarget{class_recall_exception}{}\doxysection{Recall\+Exception类 参考}
\label{class_recall_exception}\index{RecallException@{RecallException}}


{\ttfamily \#include $<$tools.\+h$>$}



类 Recall\+Exception 继承关系图\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=175pt]{class_recall_exception__inherit__graph}
\end{center}
\end{figure}


Recall\+Exception 的协作图\+:
\nopagebreak
\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[width=175pt]{class_recall_exception__coll__graph}
\end{center}
\end{figure}
\doxysubsection*{Public 成员函数}
\begin{DoxyCompactItemize}
\item 
\mbox{\Hypertarget{class_recall_exception_aa8fe3dc3b16af078242f145685082087}\label{class_recall_exception_aa8fe3dc3b16af078242f145685082087}} 
std\+::string \mbox{\hyperlink{class_recall_exception_aa8fe3dc3b16af078242f145685082087}{what}} ()
\begin{DoxyCompactList}\small\item\em 异常信息 \end{DoxyCompactList}\end{DoxyCompactItemize}


\doxysubsection{详细描述}
撤回异常 \begin{DoxySeeAlso}{参见}
\mbox{\hyperlink{class_mirai_c_p_exception}{Mirai\+CPException}} 
\end{DoxySeeAlso}
\begin{Desc}
\item[示例]\par
\mbox{\hyperlink{_d_1_2_git_2mirai_2_mirai_c_p_2cpp_2tools_8h-example}{D\+:/\+Git/mirai/\+Mirai\+CP/cpp/tools.\+h}}.\end{Desc}


该类的文档由以下文件生成\+:\begin{DoxyCompactItemize}
\item 
D\+:/\+Git/mirai/\+Mirai\+CP/cpp/tools.\+h\end{DoxyCompactItemize}
